site/docs/v1/tech/apis/applications.adoc

```diff
@@ -55,6 +55,44 @@ include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-create-operation.adoc[
 include::docs/v1/tech/apis/_application-request-body.adoc[includeRoles=true]
 :includeRoles!:
 
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create an Application from an existing Application with a generated Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/application#
+--
+
+[NOTE.since]
+====
+This API has been available since 1.43.0
+====
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-create-operation.adoc[]
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create an Application from an existing Application with the provided unique Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/application/`\{applicationId\}`#
+--
+
+[NOTE.since]
+====
+This API has been available since 1.43.0
+====
+
+==== Request Parameters
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional# [default]#defaults to a generated UUID#::
+The Id to use for the new Application. If not specified a secure random UUID will be generated.
+
+include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-create-operation.adoc[]
+
+include::docs/v1/tech/apis/_application-copy-request-body.adoc
```

Comment:
Your include is broken

Reply:
Added `[]` to the ends of this line ... seemingly still renders without.